test/unit/node/util.test.ts

```diff
@@ -0,0 +1,10 @@
+import { getEnvPaths } from "../../../src/node/util"
+
+describe("getEnvPaths", () => {
+  it("should return an object with the data, config and runtime path", () => {
+    const actualPaths = getEnvPaths()
+    expect(actualPaths.hasOwnProperty("data")).toBe(true)
+    expect(actualPaths.hasOwnProperty("config")).toBe(true)
+    expect(actualPaths.hasOwnProperty("runtime")).toBe(true)
+  })
+})
```

Comment:
Eventually, we'll add more tests. This was just to get the ball rolling.

Reply:
🤟